mbedtls/src/pk/dhparam.rs

```diff
@@ -22,6 +22,7 @@ impl Dhm {
     /// Takes both DER and PEM forms of FFDH parameters in `DHParams` format.
     ///
     /// When calling on PEM-encoded data, `params` must be NULL-terminated
+    #[allow(dead_code)]
```

Comment:
Why is this dead code? This is called from ssl::Config?

Reply:
Actually, you should make this `pub` now that you no longer call it from ssl::Config